lib/sidekiq/failures/web_extension.rb

```diff
@@ -1,50 +1,35 @@
 module Sidekiq
   module Failures
     module WebExtension
-
       def self.registered(app)
         view_path = File.join(File.expand_path("..", __FILE__), "views")
 
-        app.helpers do
-          def safe_relative_time(time)
-            return unless time
-
-            time = if time.is_a?(Numeric)
-              Time.at(time)
-            else
-              Time.parse(time)
-            end
-
-            relative_time(time)
-          end
-        end
-
         app.get "/failures" do
           @count = (params[:count] || 25).to_i
-          (@current_page, @total_size, @failures) = page(LIST_KEY, params[:page], @count, :reverse => true)
-          @failures = @failures.map {|msg, score| Sidekiq::SortedEntry.new(nil, score, msg) }
+          (@current_page, @total_size, @failures) = page(LIST_KEY, params[:page], @count, reverse: true)
+          @failures = @failures.map { |msg, score| Sidekiq::SortedEntry.new(nil, score, msg) }
 
           render(:erb, File.read(File.join(view_path, "failures.erb")))
         end
 
         app.get "/failures/:key" do
-          halt 404 unless params['key']
+          halt 404 unless params["key"]
```

Comment:
Thanks for putting these changes together! This route appears to be broken. Are you able to view specific failures with sidekiq 8?